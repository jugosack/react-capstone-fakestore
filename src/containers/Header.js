import React from 'react';
import { NavLink } from 'react-router-dom';
import mic from '../images/mic.png';
import setting from '../images/setting.png';
import '../App.css';

const Header = () => (
  <div className="ui fixed menu">
    <div className="ui container center">
      <div className="back-btn">
        <NavLink to="/">
          <img alt="Back button" src="https://img.icons8.com/ios-glyphs/30/ffffff/circled-left-2.png" />
        </NavLink>
      </div>
      <div className="headername">
        <h2 className="headerFake">FakeShop</h2>
      </div>
      <div className="position" aria-hidden="true">
        <img className="mic" src={mic} alt="mic" />
        <img className="setting" src={setting} alt="setting" />
      </div>
    </div>
  </div>
);

export default Header;
