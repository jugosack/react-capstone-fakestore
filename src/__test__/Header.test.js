import React from 'react';
import { BrowserRouter } from 'react-router-dom';
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';
import Header from '../containers/Header';

const MockHeader = () => (
  <BrowserRouter>
    <Header />
  </BrowserRouter>
);

test('renders correctly', () => {
  const tree = renderer
    .create(<MockHeader />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
