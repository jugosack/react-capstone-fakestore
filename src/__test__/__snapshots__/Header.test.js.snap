// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui fixed menu"
>
  <div
    className="ui container center"
  >
    <div
      className="back-btn"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        <img
          alt="Back button"
          src="https://img.icons8.com/ios-glyphs/30/ffffff/circled-left-2.png"
        />
      </a>
    </div>
    <div
      className="headername"
    >
      <h2
        className="headerFake"
      >
        FakeShop
      </h2>
    </div>
    <div
      aria-hidden="true"
      className="position"
    >
      <img
        alt="mic"
        className="mic"
        src="mic.png"
      />
      <img
        alt="setting"
        className="setting"
        src="setting.png"
      />
    </div>
  </div>
</div>
`;
